CHANGELOG.md

```diff
@@ -5,6 +5,24 @@ information on the list of deprecated flags and APIs please have a look at
 https://docs.docker.com/engine/deprecated/ where target removal dates can also
 be found.
 
+## 17.03.1-ce (2017-03-20)
+
+### Remote API & Client
+
+* Fix autoremove on older api [#31692](https://github.com/docker/docker/pull/31692)
+
+### Runtime
+
+* Update runc to 54296cf40ad8143b62dbcaa1d90e520a2136ddfe [#3166](https://github.com/docker/docker/pull/31666)
+ * Ignore cgroup2 mountpoints [opencontainers/runc#1266](https://github.com/opencontainers/runc/pull/1266)
+* Update containerd to 595e75c212d19a81d2b808a518fe1afc1391dad5 [#31662](https://github.com/docker/docker/pull/31662)
+ * Register healtcheck service before calling restore() [docker/containerd#609](https://github.com/docker/containerd/pull/609)
+
+### Swarm Mode
+
+* Update swarmkit to f93948cb430facd540e3c65c606384a89b3ac40f [#31742](https://github.com/docker/docker/pull/31742)
```

Comment:
Let's list the fixes from this PR instead. I'd summarize them as follows:

- Allow removal of a network when only dead tasks reference it docker/swarmkit#2018
- Retry failed network allocations less aggressively docker/swarmkit#2021
- Avoid network allocation for tasks that are no longer running docker/swarmkit#2017
- Bookkeeping fixes inside network allocator allocator: Remove deleted networks from the unallocated set docker/swarmkit#2019 docker/swarmkit#2020

Or we could list the other vendor PR instead of the SwarmKit PR below. But we should be consistent, and I think listing user-facing fixes is more useful.